passes/techmap/flatten.cc

```diff
@@ -275,6 +295,14 @@ struct FlattenPass : public Pass {
 		log("    -wb\n");
 		log("        Ignore the 'whitebox' attribute on cell implementations.\n");
 		log("\n");
+		log("    -noscopeinfo\n");
+		log("        Do not create '$scopeinfo' cells that preserve attributes of cells and\n");
+		log("        modules that were removed during flattening. With this option the\n");
+		log("        'src' attributes is handled like it was before '$scopeinfo' was\n");
```

Comment:
Nit: typos here, either "attribute is handled" or "attributes are handled", "separated"